handlers/handlers.go

```diff
@@ -14,7 +14,7 @@ import (
 )
 
 // Cookies that will not be removed deleted
-var protectedCookies = [5]string{"access_token", "lang", "collection", "timeseriesbasket", "rememberBasket"}
+var protectedCookies = [7]string{"access_token", "refresh_token", "id_token", "lang", "collection", "timeseriesbasket", "rememberBasket"}
```

Comment:
Is the size value necessary here? We are constructing and assigning in the same statement so we can don't need to specify the size as its implied/determined by the size of the array literal we're creating.
```
var protectedCookies = []string{"access_token", "refresh_token", "id_token", "lang", "collection", "timeseriesbasket", "rememberBasket"}
```

Reply:
Cheers, done